torch/csrc/distributed/rpc/testing/init.cpp

```diff
@@ -97,6 +98,11 @@ PyObject* faulty_agent_init(PyObject* _unused, PyObject* noargs) {
           (const WorkerInfo& (ProcessGroupAgent::*)(const std::string&)const) &
               ProcessGroupAgent::getWorkerInfo,
           py::call_guard<py::gil_scoped_release>())
+      .def(
```

Comment:
Have to import this function, otherwise mypy will report in class FaultyProcessGroupRpcBackendOptions, "get_worker_info" has different signature than its super class.